module/VuFind/src/VuFind/I18n/Translator/LanguageInitializerTrait.php

```diff
@@ -45,6 +44,25 @@
  */
 trait LanguageInitializerTrait
 {
+    /**
+     * Path resolver.
+     *
+     * @var PathResolver
+     */
+    protected PathResolver $pathResolver;
```

Comment:
Since we're not setting this via constructor, we ought to make it optional:

```suggestion
     * @var ?PathResolver
     */
    protected ?PathResolver $pathResolver = null;
```